Sources/LLBuildManifest/BuildManifest.swift

```diff
@@ -36,12 +36,13 @@ public struct BuildManifest {
         return result
     }
 
-    func createTarget(_ name: TargetName) -> Target {
-        Target(name: name, nodes: [])
+    public mutating func createTarget(_ name: TargetName) {
```

Comment:
This seems like a dangerous API but it seems like we might not really need it? Can we just drop it?

Reply:
How would you do otherwise?

Reply:
Depends on the use case but the fact that this can `fatalError` if clients are not careful seems dangerous. It would be better to express this requirement using the type system (if possible).

Reply:
Oh right. I misunderstood what bothered you. I can change the code to avoid a fatalError.